package.json

```diff
@@ -20,7 +20,33 @@
     "postpublish": "./publish-docs.sh",
     "prerelease": "npm test",
     "release": "standard-version -s",
-    "lint": "eslint examples/*/*.js test/*.js index.js lib/*.js"
+    "lint": "npx xo index.js lib/*.js test/*.js"
```

Comment:
do you need `npx` here?  and i don't think you need to specify the files you want to lint, i believe by default, xo will lint any `.js` file in the whole project

Reply:
Yeah, but I didn't want to include the `examples` directory for now. It's catching a LOT of stuff in the React bits that I wasn't sure how best to deal with.